### DISCUSSION TITLE
How to edit the date that appears in Plex ?

### DISCUSSION THREAD

**Discussion by NightMayor1** [seanap/Plex-Audiobook-Guide#94] [Q&A] [answered]
Hello,
Thank you for the guide, it helped immensely.
I set everything up in Plex and if I understand correctly Audnexus agent uses the Audiobook release date instead of the book publication date (e.g. The Hobbit shows up as 2012).
![image](https://user-images.githubusercontent.com/125417730/218897341-7573c7da-d4fd-4ad5-8bf6-d09782eabbc8.png)
I found by checking the "prefer local metadata" option it would show the correct date (and use the artwork included in the tag).
![image](https://user-images.githubusercontent.com/125417730/218898016-5244bedf-8de2-40d4-a7ad-9be87e384893.png)

But even with this workaround I still have a pet peeve that if a book is translated, Plex shows the publication date of the translation instead of the original book (i.e. The Last Wish shows up as 2008 instead of 1993).
![image](https://user-images.githubusercontent.com/125417730/218899118-93a3c62f-8ba9-4f27-ab95-b7f631f953d1.png)

I tried changing the tags, titles and folder name manually and changing every date to 1993, but plex still shows 2008. 
![image](https://user-images.githubusercontent.com/125417730/218899459-1c7b3c96-6ebe-469f-8f01-4247b3633d7c.png)

Any ideas how I can change this date to show up to what I want in Plex ?
Thank you.

**Comment by rabelux** [comment, marked as answer]
If you set Plex to prefer local metadata you have to do a "rescan" of your library instead of a metadata refresh. Hope that helps.

By the way, audnexus has a setting `Use copyright year instead of datePublished` which might solve a few of your cases but probably not all.

**Reply by NightMayor1** [reply]
Thank you for the reply. Copyright year still wouldn't do it for some audiobooks like Shahnameh.
So I did a bit more testing and found local metadata and manually editing the year does what I'm looking for, but what had me confused was that it didn't work for some of the audiobooks I had already added to plex and had edited after the fact, no matter how many times I refreshed the metadata or rescanned.  
Every new audiobook I add with the corrected date works perfectly with the above method.
